paper/basic_training.tex

```diff
@@ -488,13 +495,76 @@ \subsubsection{Online resources}
 \section{Basic simulation concepts and terminology}
 
 \subsection{Force fields}
+\label{sec:force_fields}
 \begin{itemize}
 \item What are they?
 \item Functional forms for various terms
 \item Common force fields
 \item Limitations
 \end{itemize}
 
+The term "force field" simply refers to the included terms, particular form,
+and specific implementation details, including parameter values, of the 
+chosen potential energy function. Most of the terms included in potential energy functions have already been detailed in Section~\ref{sec:mol_interactions}, with the most common being Coulombic, Lennard-Jones, bond, angle, and torsional (dihedral) terms. 
+Here, we very briefly describe the mathematical forms used to represent such interactions.
+
+Non-bonded interactions of the Lennard-Jones form are well-described 
+throughout the literature (for instance see Ch. 4 of \citet{LeachBook}). 
+Coulombic interactions, including both short and long-range components, are 
+described in detail elsewhere in this document.
+To represent bonded interactions, harmonic potentials are often employed. 
+The same is true for angles between three bonded atoms, but the harmonic potential 
+is applied with respect to the angle formed and not the distance between atoms. 
+Torsional terms are also commonly employed, usually consisting as sums of 
+cosines. More exotic potentials based on three-body intermolecular 
+orientations, or terms directly coupling bond lengths and bending angles are 
+also possible. Additionally, other choices of potential function are of course 
+acceptable, including Buckingham or Morse potentials, or the use of "improper" 
+dihedral terms to enforce planarity of cyclic portions of molecules. 
+%JIM: Not sure if it's important, but I wanted to include something
+%about force field terms like CMAP, as well as external fields. The 
+%below sentence is an attempt at that, but it may not be needed.
+This may even include empirical corrections based on discrete binning 
+along a particular set of degrees of freedom, as well as applied external  
+fields (i.e. electric fields) and force field terms describing the effect of 
+degrees of freedom, such as solvent, that have been removed from the system via 
+"coarse-graining." For a more in-depth discussion of common (and even less common) force field terms, see Ch. 4 of \citet{LeachBook}.
+
+Functional forms used to describe specific terms in a potential energy function 
```

Comment:
Not sure I follow what you're getting at here in this sentence beginning "Functional forms..."; is this comparing different force field families? Maybe an example would help?